### PR TITLE
Add further questions

### PR DESCRIPTION
**Pull request by patch0**
* Re-orders the questions to match demo
* Add images (ish)

### CODE REVIEW

File: en/quiz1/question_1.md [hunk 1]
```diff
@@ -9,74 +9,49 @@ A question may optionally have a 'frontmatter' section with the `legend` paramet
...
 ---
-legend: Question 1 of 2
+legend: Question 1 of 3
 ---
 
-The bug sprite starts facing upwards.
+A dog sprite in Scratch has the following code:
 
-![Bug facing upwards](./img/q3-1.png)
+![A dog](images/quiz/q1-1.png) ![Selection of four scratch blocks](images/quiz/q1-2.png)
```
Comment (glenpike) [inline en/quiz1/question_1.md:17]: needs to be '../images/quiz/q1-2.png'? etc. for other images?  If I view this file in the PR images are broken

Comment (patch0) [reply en/quiz1/question_1.md:17]: Yeah, but I'm not sure if the processing will magically fix? @sHtev ?

Comment (glenpike) [reply en/quiz1/question_1.md:17]: If you preview the other projects like Rock Band, the images in GH show up.  I don't think we do anything in Kramdown that's custom for images?

**Review by glenpike — APPROVED** [review]
Test on staging

**Pull request merged**